installer/resources/pacbot_app/files/DB.sql

```diff
@@ -3347,6 +3347,8 @@ CALL alter_cf_AssetGroupDetails_alter_user_length();
 
 UPDATE cf_Target SET displayName='PostgreSQL' WHERE targetName in ('cloudsql_postgres','postgresql');
 
+update pac_config_properties set application = 'application' where cfkey = 'aws.enabled';
```

Comment:
@santhosh-challa please see this update statement

Reply:
This LGTM. We need to be careful on any updates on policy tables.This is fine.

We can remove this update post release or need to find a way to execute the all updates only once.